comp/core/secrets/secretsimpl/secrets.go

```diff
@@ -356,6 +322,22 @@ func (r *secretResolver) resolve(data []byte, origin string, notifyCb secrets.Re
 		if err := w.walk(&config); err != nil {
 			return nil, err
 		}
+
+		// notify subscriptions about the changes to secrets
+		for handle, newValue := range secretResponse {
+			oldValue := r.cache[handle]
+			for _, ctx := range r.origin[handle] {
```

Comment:
Since `ctx` is a common standard variable name that we tend to associate with `context.Context` why don't we rename this variable to `secretCtx`?